packages/backend/.env.example

```
@@ -26,8 +26,17 @@ ERROR_HANDLING_ENABLE_RATE_LIMITING=true
 # ==== Logging Configuration ====
 LOG_LEVEL=info
 
-# Redis Configuration
-REDIS_URL=redis://localhost:6379
+# ==== Cache Configuration ====
+# TTL in Sekunden (3600 = 1 Stunde default)
+CACHE_TTL_SECONDS=3600
+# Maximale Anzahl von Cache-Einträgen
+CACHE_MAX_ITEMS=1000
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The CACHE_MAX_ITEMS key should go before the CACHE_TTL_SECONDS key

(UnorderedKey)